tools/accuracy_checker/datasets_preparation.md

```diff
@@ -0,0 +1,62 @@
+# Datasets preparation
+
+If you want to use prepared configs to run the Accuracy Checker tool and the Model quantizer, you need to organize folders with validation datasets in a certain way.  Instructions for preparing validation data are described in this document.
+
+## [ImageNet](http://image-net.org)
+
+### How download dataset
+
+To download images from ImageNet, you need to have an account and agree to the Terms of Access. Follow the steps below:
+1. Go to the [ImageNet](http://www.image-net.org/) homepage
+2. If you have an account, click `Login`. Otherwise, click `Signup` in the right upper corner, provide your data, and wait for a confirmation email
+3. Log in after receiving the confirmation email and go to the `Download` tab
+4. Select `Download Original Images`
+5. You will be redirected to the Terms of Access page. If you agree to the Terms, continue by clicking Agree and Sign
+6. Click one of the links in the Download as one tar file section to select it
+7. Unpack archive
+
+### Structure of directory with dataset
+
+This section describes what structure of directory dataset must have to run the Accuracy Checker tool.
+`<DATASET_DIR>` must contain the following entries:
+* A subdirectory named `ILSVRC2012_img_val` containing the ILSVRC 2012 validation images. How to download them is described above.
+* One of the annotation files:
+    * `val.txt` from unpacked archive <http://dl.caffe.berkeleyvision.org/caffe_ilsvrc12.tar.gz>.
+    * `ILSVRC2017_val.txt` from <https://raw.githubusercontent.com/hujie-frank/SENet/master/ILSVRC2017_val.txt> which must be renamed to `val15.txt`
```

Comment:
I suggest more directory-like approach (the main idea - put entity name in the beginning):
```
* `<DATASET_DIR>` - root directory
    * `ILSVRC2012_img_val` - contains the ILSVRC 2012 validation images
    * `val.txt` - annotation file 
    * `val15.txt` - annotation file 
```
which looks as:
* `<DATASET_DIR>` - root directory
    * `ILSVRC2012_img_val` - contains the ILSVRC 2012 validation images
    * `val.txt` - annotation file 
    * `val15.txt` - annotation file 

Also, you should add how-to get annotation files to *How download dataset* section.
Question: what is the difference between `val.txt` and `val15.txt`?

Reply:
val.txt is used for imagenet 2012
val15.txt is used for imagenet 2015
The difference in annotation files is the labels order

Reply:
Please review again